gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -1899,10 +1899,11 @@ def inventoryHash_locked
       @cacheLock.sync_lock(:EX) if (unlock = @cacheLock.sync_shared?)
 
       $vim_log.info "MiqVimInventory(#{@server}, #{@username}).inventoryHash_locked: calling retrieveProperties" if $vim_log
-      rv = retrieveProperties(@propCol, @spec)
+      rra = retrievePropertiesEx(@propCol, @spec)
+      oca = rra['objects'] if rra
```

Comment:
I like `retrievePropertiesCompat`, originally I had `retrievePropertiesEx` return `.fetch_path('returnval', 'objects')` but that would prevent callers from actually using the additional things that `RetrievePropertiesEx` returned.  This seems much cleaner.